packages/icons/icons.json

```
@@ -3075,644 +3075,644 @@
         "iconName": "notifications-snooze",
-        "tags": "notification, information, message, snooze, suppress",
+        "tags": "information, message, suppress",
         "group": "interface",
         "codepoint": 62044
     },
     {
         "displayName": "High Priority",
         "iconName": "high-priority",
-        "tags": "notification, message",
+        "tags": "notification, message, loud, stop, emergency, exclamation, alert, attention, exclamation mark, !, warning, error, exclamation point, bang",
         "group": "interface",
         "codepoint": 61934
     },
     {
         "displayName": "Emoji",
         "iconName": "emoji",
-        "tags": "message, chat, reaction",
+        "tags": "message, chat, reaction, happy, smile, grin, smiley, happy face, excitement",
         "group": "interface",
         "codepoint": 61846
     },
     {
         "displayName": "Add Location",
         "iconName": "add-location",
-        "tags": "location, add,",
+        "tags": "location, new, plus, object object, node, circle",
         "group": "interface",
         "codepoint": 61700
     },
     {
         "displayName": "Shapes",
         "iconName": "shapes",
-        "tags": "shapes, geometric,",
+        "tags": "geometric, building blocks",
         "group": "interface",
         "codepoint": 62128
     },
     {
         "displayName": "Shared Filter",
         "iconName": "shared-filter",
-        "tags": "notification, filter, shared",
+        "tags": "notification, contacts, funnel, people",
         "group": "interface",
         "codepoint": 62130
     },
     {
         "displayName": "One to one",
         "iconName": "one-to-one",
-        "tags": "object, relation, ontology",
+        "tags": "object, relation, ontology, link, connect, join, on, edge, nodes",
         "group": "data",
         "codepoint": 62054
     },
     {
         "displayName": "One to many",
         "iconName": "one-to-many",
-        "tags": "object, relation, ontology",
+        "tags": "object, relation, ontology, branch, fork, split, usb, network, lineage",
         "group": "data",
         "codepoint": 62053
     },
     {
         "displayName": "Many to one",
         "iconName": "many-to-one",
-        "tags": "object, relation, ontology",
+        "tags": "object, relation, ontology, link, connect, share, merge, flow",
         "group": "data",
         "codepoint": 62008
     },
     {
         "displayName": "Many to many",
         "iconName": "many-to-many",
-        "tags": "object, relation, ontology",
+        "tags": "object, relation, ontology, graph, cross, nodes",
         "group": "data",
         "codepoint": 62007
     },
     {
         "displayName": "Geometry: Stadium",
         "iconName": "stadium-geometry",
-        "tags": "geometry, shape, race track, ellipse, discorectangle, obround",
+        "tags": "shape, race track, ellipse, round, oval, loop, track, home button",
```

actually these existing tags were intentional and precise terms for this shape:

```suggestion
        "tags": "shape, race track, ellipse, discorectangle, obround, round, oval, loop, track, home button",
```